format-sdk/src/main/cpp/jni_wrapper.cpp

```diff
@@ -0,0 +1,171 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <jni.h>
+#include <string>
+#include <string_view>
+
+#include <arrow/status.h>
+
+#include "format/starrocks_format_writer.h"
+#include "jni_utils.h"
+#include "starrocks_format/starrocks_lib.h"
+
+namespace starrocks::lake::format {
+
+#ifdef __cplusplus
+    extern "C" {
+#endif
+
+    static jint jniVersion = JNI_VERSION_1_8;
+
+    jclass kNativeOptExceptionClass;
+
+    jint JNI_OnLoad(JavaVM *vm, void *reserved) {
+        JNIEnv *env;
+        if (vm->GetEnv(reinterpret_cast<void **>(&env), jniVersion) != JNI_OK) {
+            return JNI_ERR;
+        }
+
+        kNativeOptExceptionClass = find_class(env, "Lcom/starrocks/format/jni/NativeOperateException;");
```

Comment:
I thought FindClass does not need `Lxxx;` in `"Lcom/starrocks/format/jni/NativeOperateException;"` , should  you double confirm this?

Reply:
I checked the [Oracle JNI docs](https://docs.oracle.com/javase/8/docs/technotes/guides/jni/spec/functions.html#FindClass), and the `Lxxx` here is not needed. I have removed it.